diff --git a/os.module b/os.module
index de19b64f..d59beec0 100755
--- a/os.module
+++ b/os.module
@@ -641,12 +641,15 @@ function os_process_node(&$vars) {
   $vars['theme_hook_suggestions'][] = 'node__'.$vars['type'].'__'.$vars['view_mode'];
 
   // Building "Read more" link for "No image teaser" for profiles
-  if($vars['type'] == 'person' && $vars['content']['pic_bio']['body']['#view_mode'] == 'no_image_teaser') {
-    $vars['content']['links']['node']['#links']['node-readmore'] = array(
-                                                                      'title' => t('Read more'),
-                                                                      'href' => 'node/' . $vars['nid'],
-                                                                      'attributes' => array('rel' => 'tag', 'title' => strip_tags($vars['title']))
-                                                                    );
+  if($vars['type'] == 'person' 
+      && (isset($vars['content']['pic_bio']['body']['#view_mode'])) 
+      && $vars['content']['pic_bio']['body']['#view_mode'] == 'no_image_teaser') {
+    $vars['content']['links']['node']['#links']['node-readmore'] 
+        = array(
+            'title' => t('Read more'),
+            'href' => 'node/' . $vars['nid'],
+            'attributes' => array('rel' => 'tag', 'title' => strip_tags($vars['title']))
+          );
   }
   // Only proceeds if we have a "Read more" link.
   if (!isset($vars['content']['links']['node']['#links']['node-readmore'])) {
