diff --git a/os_publications.module b/os_publications.module
index 568da52e..63115b6c 100644
--- a/os_publications.module
+++ b/os_publications.module
@@ -1006,7 +1006,7 @@ function os_publications_entity_prepare_view($entities, $type, $langcode) {
 
       // Avoid redundancy in publication label, e.g., "United States patent US 123".
       // If country code and country name are both displayed, remove the country code.
-      $biblio_style = $user->data['biblio_user_style'];
+      $biblio_style = isset($user->data['biblio_user_style']) ? $user->data['biblio_user_style'] : NULL;
       if (($biblio_style == null) or ($biblio_style == "system")) {
         $biblio_style = variable_get('biblio_citeproc_style');
       }
